### PR TITLE
P0 questions result

### PR DESCRIPTION
**Pull request by itsmebipul00**
Results Page and Questions Page Designed

### CODE REVIEW

**Review by Megha-Joshi — APPROVED** [review]
Nice use of variables throughout the code. Good formatting. Ready to merge.

File: style.css [hunk 1]
```diff
@@ -29,7 +33,7 @@ nav {
 
 .primary-navigation {
   display: flex;
```
Comment (sach10-create) [inline style.css:35]: can be clubbed into a class, as it is used many times

**Review by sach10-create — APPROVED** [review]
Great work with classes.

**Review by Shubhamkr2610 — APPROVED** [review]
Code looks clean. Everything looks good to me. You are good to go.

**Review by Shubhamkr2610 — APPROVED** [review]
Everything is food and fine. You can merge.

File: style.css (hunk 1 above)
Comment (itsmebipul00) [reply style.css:35]: Thank you for pointing it! Overlooked this

**Pull request merged**